src/main/java/projectbookstore/dto/category/CreateCategoryRequestDto.java

```diff
@@ -2,9 +2,11 @@
 
 import jakarta.validation.constraints.NotBlank;
 import jakarta.validation.constraints.NotNull;
-import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
 
-@Data
+@Getter
+@Setter
```

Comment:
why not data? DTO classes also need `equals`, `hashcode` & `toString` methods

Reply:
Sorry !! I had lots of conflicts after rebase and this happened again((